lib/bambora/v1/payment_resource.rb

```diff
@@ -3,15 +3,32 @@
 module Bambora
   module V1
     class PaymentResource
+      CREDIT_CARD_ID_DEFAULT = 1
```

Comment:
Is this `1` because a payment profile could have multiple credit cards associated with it but in practice never does for our use case?

Reply:
Yes exactly.